hilti/runtime/include/types/stream.h

```diff
@@ -105,24 +112,28 @@ class Chunk {
         _offset = other._offset;
         _data = std::move(other._data);
         _next = std::move(other._next);
-        _chain = std::move(other._chain);
+        _chain = other._chain;
         return *this;
     }
 
     ~Chunk() = default;
 
     Offset offset() const { return _offset; }
     Offset endOffset() const { return _offset + size(); }
-    bool isCompact() const { return std::holds_alternative<Array>(_data); }
+    bool isCompact() const { return ! std::holds_alternative<Vector>(_data); }
```

Comment:
According to a quick grep, it doesn't look like we're using this method anywhere. Given that the semantics are getting a bit odd now, I'd just remove it.